sections/queueing_model/subsections/performance_measures/blocking_time/main.tex

```diff
@@ -0,0 +1,125 @@
+    c(u,v) = 
+    \begin{cases}
+        \frac{1}{\min(v,C) \mu}, & \text{if } v = N\\
+        \frac{1}{\lambda_1 + \min(v,C) \mu}, & \text{otherwise}
+    \end{cases}
+\end{equation}
+
+\begin{equation}\label{eq:probs_of_service_and_arrival}
+    p_s(u,v) = \frac{\min(v,C)\mu}{\lambda_1 + \min(v,C)\mu}, \qquad
+    p_a(u,v) = \frac{\lambda_1}{\lambda_1 + \min(v,C)\mu}
+\end{equation}
+
+% TODO: These might possibly need to be moved to the appendix 
+The system of equations produced by (\ref{eq:general_blocking_time_at_each_state}) can be solved by considering the linear system \(Zx=y\), where:
+
+\begin{equation}\label{eq:general_algebaric_approach_blocking_time}
+    \scalebox{0.73}{\(
+        Z = 
+        \begin{pmatrix}
+            -1 & p_a & 0 & \dots & 0 & 0 & 0 & 0 & 0 & \dots & 0 & 0 \\ %(1,T)
+            p_s & -1 & p_a & \dots & 0 & 0 & 0 & 0 & 0 & \dots & 0 & 0 \\ %(1,T+1)
+            0 & p_s & -1 & \dots & 0 & 0 & 0 & 0 & 0 & \dots & 0 & 0 \\ %(1,T+2)
+            \vdots & \vdots & \vdots & \ddots & \vdots & \vdots & \vdots & 
+            \vdots & \vdots & \ddots & \vdots & \vdots \\ 
+            0 & 0 & 0 & \dots & 1 & -1 & 0 & 0 & 0 & \dots & 0 & 0 \\ %(1,N)
+            p_s & 0 & 0 & \dots & 0 & 0 & -1 & p_a & 0 & \dots & 0 & 0 \\ %(2,T)
+            0 & 0 & 0 & \dots & 0 & 0 & p_s & -1 & p_a & \dots & 0 & 0 \\ %(2,T+1)
+            \vdots & \vdots & \vdots & \ddots & \vdots & \vdots & \vdots & 
+            \vdots & \vdots & \ddots & \vdots & \vdots \\ 
+            0 & 0 & 0 & \dots & 0 & 0 & 0 & 0 & 0 & \dots & 1 & -1 \\ %(M,N)
+        \end{pmatrix},
+        x = 
+        \begin{pmatrix}
+            b(1,T) \\
+            b(1,T+1) \\
+            b(1,T+2) \\
+            \vdots \\
+            b(1,N) \\
+            b(2,T) \\
+            b(2,T+1) \\
+            \vdots \\
+            b(M,N) \\
+        \end{pmatrix}, 
+        y= 
+        \begin{pmatrix}
+            -c(1,T) \\
+            -c(1,T+1) \\
+            -c(1,T+2) \\
+            \vdots \\
+            -c(1,N) \\
+            -c(2,T) \\
+            -c(2,T+1) \\
+            \vdots \\
+            -c(M,N) \\
+        \end{pmatrix}
+    \)}
+\end{equation}
+
+Matrix \(Z\) can be alternatively defined using \(Z_{ij}\) defined in equation 
+(\ref{eq:general_mapping_function_of_blocking_matrix}) where \(i\) and \(j\) 
+represent states \((u_i, v_i), (u_j, v_j) \in S_b\).
+
+\begin{equation}\label{eq:general_mapping_function_of_blocking_matrix}
+    Z_{ij} = 
+    \begin{cases}
+        p_a, & \textbf{if } j = i + 1 \textbf{ and } v_i \neq N \\
+        \\
+        p_s, & \textbf{if } j = i - 1 \textbf{ and } v_i \neq N, v_i \neq T \\
+             & \textbf{or } j = i - N + T \textbf{ and } u_i \geq 2,\,v_i = T \\
+        \\
+        1, & \textbf{if } j = i - 1 \textbf{ and } v_i = N \\
+        \\
+        -1, & \textbf{if } i = j \\
+        \\
+        0, & \textbf{otherwise} \\
+    \end{cases}
+\end{equation}
```

Comment:
```suggestion
The system of equations produced by (\ref{eq:general_blocking_time_at_each_state}) can be solved by considering the linear system \(Zx=y\). Assuming \(i\) and \(j\) 
represent states \((u_i, v_i), (u_j, v_j) \in S_b\) then \(Z_{ij}\) is given by:

\begin{equation}\label{eq:general_mapping_function_of_blocking_matrix}
    Z_{ij} = 
    \begin{cases}
        p_a, & \textbf{if } j = i + 1 \textbf{ and } v_i \neq N \\
        \\
        p_s, & \textbf{if } j = i - 1 \textbf{ and } v_i \neq N, v_i \neq T \\
             & \textbf{or } j = i - N + T \textbf{ and } u_i \geq 2,\,v_i = T \\
        \\
        1, & \textbf{if } j = i - 1 \textbf{ and } v_i = N \\
        \\
        -1, & \textbf{if } i = j \\
        \\
        0, & \textbf{otherwise} \\
    \end{cases}
\end{equation}

Equation~(\ref{eq:general_algebaric_approach_blocking_time}) shows this.

\begin{equation}\label{eq:general_algebaric_approach_blocking_time}
    \scalebox{0.73}{\(
        Z = 
        \begin{pmatrix}
            -1 & p_a & 0 & \dots & 0 & 0 & 0 & 0 & 0 & \dots & 0 & 0 \\ %(1,T)
            p_s & -1 & p_a & \dots & 0 & 0 & 0 & 0 & 0 & \dots & 0 & 0 \\ %(1,T+1)
            0 & p_s & -1 & \dots & 0 & 0 & 0 & 0 & 0 & \dots & 0 & 0 \\ %(1,T+2)
            \vdots & \vdots & \vdots & \ddots & \vdots & \vdots & \vdots & 
            \vdots & \vdots & \ddots & \vdots & \vdots \\ 
            0 & 0 & 0 & \dots & 1 & -1 & 0 & 0 & 0 & \dots & 0 & 0 \\ %(1,N)
            p_s & 0 & 0 & \dots & 0 & 0 & -1 & p_a & 0 & \dots & 0 & 0 \\ %(2,T)
            0 & 0 & 0 & \dots & 0 & 0 & p_s & -1 & p_a & \dots & 0 & 0 \\ %(2,T+1)
            \vdots & \vdots & \vdots & \ddots & \vdots & \vdots & \vdots & 
            \vdots & \vdots & \ddots & \vdots & \vdots \\ 
            0 & 0 & 0 & \dots & 0 & 0 & 0 & 0 & 0 & \dots & 1 & -1 \\ %(M,N)
        \end{pmatrix},
        x = 
        \begin{pmatrix}
            b(1,T) \\
            b(1,T+1) \\
            b(1,T+2) \\
            \vdots \\
            b(1,N) \\
            b(2,T) \\
            b(2,T+1) \\
            \vdots \\
            b(M,N) \\
        \end{pmatrix}, 
        y= 
        \begin{pmatrix}
            -c(1,T) \\
            -c(1,T+1) \\
            -c(1,T+2) \\
            \vdots \\
            -c(1,N) \\
            -c(2,T) \\
            -c(2,T+1) \\
            \vdots \\
            -c(M,N) \\
        \end{pmatrix}
    \)}
\end{equation}


```